### PR TITLE
Sec5 bpi middleware and integration & sec 8 standards conformance

### PR DESCRIPTION
**Pull request by Therecanbeonlyone1969**
[5  Middleware, Communication and Interoperability](#5-Middleware,-Communication-and-Interoperability) \
&nbsp;&nbsp;&nbsp;&nbsp;[5.1 Resolvable Identifiers for BPI Interoperability Services](#51-Resolvable-Identifiers-for-BPI-Interoperability-Services) \
&nbsp;&nbsp;&nbsp;&nbsp;[5.2 Discoverable Authentication and Authorization Capabilities for BPI Interoperability Services](#52-Discoverable-Authentication-and-Authorization-Capabilities-for-BPI-Interoperability-Services) \
&nbsp;&nbsp;&nbsp;&nbsp;[5.3 Discoverable and Negotiable BPI Interoperability Services](#53-Discoverable-and-Negotiable-BPI-Interoperability-Services) \
&nbsp;&nbsp;&nbsp;&nbsp;[5.4 Bi- and Multi-directional and Mono-directional BPI Interoperability Services](#54-Bi-and-Multi-directional-and-Mono-directional-BPI-Interoperability-Services) \
&nbsp;&nbsp;&nbsp;&nbsp;[5.5 Standardized Set of BPI Interoperability APIs](#55-Standardized-Set-of-BPI-Interoperability-APIs)

[8 Conformance](#8-conformance) \
&nbsp;&nbsp;&nbsp;&nbsp;[8.1 Conformance Targets](#81-conformance-targets) \
&nbsp;&nbsp;&nbsp;&nbsp;[8.2 Conformance Levels](#82-Conformance-Levels)

Please, review and comment. Note, Section 5.4 is quite extensive.

### CODE REVIEW

File: core/baseline-core-v1.0-psd01.md [hunk 1]
```diff
@@ -566,678 +637,1941 @@ Below we list and define the components of each layer. The detailed requirements
...
+**[R75]**	A BPI User MUST be capable of performing cryptographic digital signature creation and verification based on BPI specified cryptographic algorithms.
 
-**[RXX]** A workgroup MUST consist of at least one participant.
+**[R76]**	A BPI User MUST be capable of performing cryptographic multi-digital signature creation and verification based on BPI specified cryptographic algorithms.
 
-Note that a workgroup participant may be a user of another BPI than the BPI the workgroup is defined in. The appropriate authentication and authorization policies to enable such a scenario are up to implementers and beyond the scope of this document.
+**[R77]**	A BPI User MUST be capable of performing cryptographic secret sharing based on BPI specified cryptographic algorithms.
 
-**[RXX]** A workgroup MUST have at least one administrator.
+An example of cryptographic secret sharing is called ["Shamir Secret Sharing"](####Shamir).
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:961]: Link broken

File: core/baseline-core-v1.0-psd01.md [hunk 2]
```diff
@@ -566,678 +637,1941 @@ Below we list and define the components of each layer. The detailed requirements
...
 
-**[RXX]** A workgroup MUST have at least one security policy.
+**[R78]** An external BPI User MUST at least be able to create, read, update and delete the following BPI core components following business rules for each component that were established by the BPI operator and agreed to by BPI Users:
+* A BPI Account belonging to the BPI User. Note that a BPI Account as defined in Section 6 is different from a BPI User account as defined in [Section 5.2](#52-BPI-User-Account).
+* A BPI Workgroup as defined in [Section 6](#-6-Agreement-Execution). 
+* A BPI Workflow as defined in [Section 6](#-6-Agreement-Execution).
+* A BPI Workstep as defined in [Section 6](#-6-Agreement-Execution).
+* A BPI Transaction as defined in [Section 6](#-6-Agreement-Execution).
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:968]: all same links

File: core/baseline-core-v1.0-psd01.md [hunk 3]
```diff
@@ -566,678 +637,1941 @@ Below we list and define the components of each layer. The detailed requirements
...
+**[R80]** A BPI User MUST be able to send and receive BPI messages from other BPI users using the BPI Communication capability as defined in [Section 5.3](#53-BPI-Communication).
 
-**[RXX]** A workgroup administrator MUST be able to perform at minimum the following functions:
-* add or remove one or more participants
-* create, update and delete both security and privacy policies.
-* delete or archive a workgroup
+**[R81]** A BPI User MUST be able to create, read BPI messages from other BPI users using the BPI Communication capability as defined in [Section 5.3](#53-BPI-Communication).
 
-**[OXX2]** A workgroup MAY have more than one administrator.
+Note that when a BPI User executes these capabilities they are understood to be enabled by services under the direct custody of the BPI User. However, the BPI User may delegate these capabilities to the BPI the BPI user operates in, called a custodial approach.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:978]: It would be easier to read to improve `to the BPI the BPI user operates`. Thanks :)

Comment (mehranshakeri) [reply core/baseline-core-v1.0-psd01.md:978]: `to the BPI, in which the BPI user operates in`

File: core/baseline-core-v1.0-psd01.md [hunk 4]
```diff
@@ -566,678 +637,1941 @@ Below we list and define the components of each layer. The detailed requirements
...
+Note that when a BPI User executes these capabilities they are understood to be enabled by services under the direct custody of the BPI User. However, the BPI User may delegate these capabilities to the BPI the BPI user operates in, called a custodial approach.  
 
-**[CRXX]>[OXX2]** There MUST a consensus model for administrative actions.
+**[R82]** A BPI User MUST be able to delegate one or more BPI User capabilities to a 3rd party that is also a BPI User.
 
-**[OXX3]** A workgroup MAY be attached to one or more workstep instances. 
+## 5.2 BPI User Account
 
-**[OXX4]** A Workgroup MAY be attached to one or more workflow instances.
+A BPI User Account is a key component of a BPI because it is the anchor point for all important BPI functions -- authentication and authorization, messages, workgroups, workflows, worksteps, transactions etc. Without the notion of a cryptographically verifiable state of a BPI User within a BPI which is afforded by a BPI User Account, no other BPI functions will be able to operate successfully since all functions are predicated on cryptographically verifiable counterparties that maintain a referable and verifiable state within a BPI.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:984]: two missing commas.

Comment (mehranshakeri) [reply core/baseline-core-v1.0-psd01.md:984]: `- which is afforded by a BPI User Account -`

File: core/baseline-core-v1.0-psd01.md [hunk 5]
```diff
@@ -566,678 +637,1941 @@ Below we list and define the components of each layer. The detailed requirements
...
 
-**[CRXX]>[OXX4]** A workgroup attached to a workflow MUST be also attached to each workstep in the workflow.
+The following requirements on a BPI User Account assume that all identifiers and credentials referring to a BPI User are compliant with the requirement in [Section 3](#3-Identifiers-Identity-and-Credential-Management).
 
-## 6.4 BPI Account
-Before we can move on to the requirements on the individual components in the processing layer required for agreement execution, we need to define and specify the prerequisites. Since we have been defining and discussing state objects in the context of a BPI, we need to define stateful object processing. This necessitates an account based model for both BPI users and BPI state objects. This is analogous to the Ethereum model using accounts for individual participants and smart contracts -- both are stateful objects.
+**[R83]** A BPI User Account MUST NOT contain any personal identifiable information (PII).
 
-**[RXX]** Each BPI user MUST be associated with an account.
+This requirement facilitates compliance with privacy laws in different jurisdiction.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:990]: missing `s`: `jurisdictions`

File: core/baseline-core-v1.0-psd01.md [hunk 6]
```diff
@@ -566,678 +637,1941 @@ Below we list and define the components of each layer. The detailed requirements
...
+**[R84]** A BPI User Account MUST NOT be created by an external BPI User.
 
-An account itself is defined through the following requirements:
+This requirement ensure that only a BPI operator can create a BPI User Account to avoid account creation spamming and implementation of KYC processes required in certain jurisdictions.
 
-**[RXX]** An account MUST have a unique account number.
+**[R85]** A BPI User Account MUST have at least the following data properties:
+1. A unique, resolvable and cryptographically verifiable identifier
+2. One or more security policies including authentication and authorization policies
+3. A cryptographically verifiable credential establishing the (legal) identity of a BPI User and utilized by a BPI in the creation of the BPI Usr Account
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:999]: `User` instead of `Usr`

File: core/baseline-core-v1.0-psd01.md [hunk 7]
```diff
@@ -566,678 +637,1941 @@ Below we list and define the components of each layer. The detailed requirements
...
+4. A BPI User Account recovery key that can be only derived by the BPI User owning the BPI User Account and is independent from the private key(s) associated with the BPI User Account identifier.
+5. A list of all BPI Accounts related to the BPI User Account
 
-**[RXX]** An account MUST have at least one account owner.
+This minimal set of requirements ensures:
+* that access to a BPI User account can be cryptographically verified (bullet 1),
+* that a BPI account can fine grain access and authorization requirements for the BPI User (bullet 2),
+* that other BPI User can independently verify the identity used to establish the BPI User Account (bullet 3),
+* that a BPI User can independently recover a BPI User Account even if their private key was compromised, and ensure forward security (bullet 4), and
+* that there is a provable relationships with BPI Accounts connected to BPI state objects connected to the BPI User Account owner (bullet 5).
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1008]: `a provable relationship` without `s`

File: core/baseline-core-v1.0-psd01.md [hunk 8]
```diff
@@ -566,678 +637,1941 @@ Below we list and define the components of each layer. The detailed requirements
...
+This is a critical audit requirement to ensure ease of compliance with regulatory statutes across jurisdictions.
 
-An account authorization condition is the condition which have to be met by the account owners to authorize a BPI transaction from that account.
+**[R87]** A BPI User Account MUST be encrypted to and decrypted by one or more cryptographic keys only known to the BPI User Account holder.
 
-**[RXX]** Account ownership and associated authorizations MUST be cryptographically provable.
+**[D13]** A BPI User Account SHOULD be re-encrypted after every BPI User session to one or more cryptographic keys only known to the BPI User Account holder.
 
-This may be achieved through for example a cryptographic digital signature.
+This requirement and recommendation ensure that a compromise of a BPI does not expose BPI User Account information. A BPI may use the BPI User Account unique identifier or another unique identifer such as an account number known to the BPI User Account holder as an unencrypted identifier of the BPI User Account stored within the BPI.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1018]: Can mention [R87] & [D13] explicitly.

File: core/baseline-core-v1.0-psd01.md [hunk 9]
```diff
@@ -566,678 +637,1941 @@ Below we list and define the components of each layer. The detailed requirements
...
 
-**[RXX]** Each transaction MUST contain a representation of the (commercial) document as a state object constituting the suggested new agreement state, such that it can be validated by the prover system associated with the account of the state object representing the agreement state to be altered.  
+This requirement ensures that overall system latency is not impacted when volume meaningfully and rapidly changes.
 
-**[RXX]** If there is more than one prover system associated with the receiver account, the transaction MUST unambiguously identify which prover system the transaction is targeting.
+## 5.3 BPI Communication
 
-**[RXX]** Each transaction MUST contain the cryptographic digital signature of the owner(s) of the Sender account.
+The BPI Communication capability is a foundational element of any BPI in order to facilitate complex workflows in workgroups. Therefore, it is supposed to meet the semantically demanding workflow communication and workflow orchestration requirements between BPI User as well as the complex and highly asynchronous nature of BPI transactions originating from such workflows. This leads to the following core capability requirements:
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1068]: s missing `between BPI Users as well`

Comment (mehranshakeri) [reply core/baseline-core-v1.0-psd01.md:1068]: Improve with bulletpoints

File: core/baseline-core-v1.0-psd01.md [hunk 10]
```diff
@@ -566,678 +637,1941 @@ Below we list and define the components of each layer. The detailed requirements
...
+**[R103]** BPI communication protocols MUST be message-based.
 
-Note, that this is only a minimal set of requirements on an invalid transaction. Each BPI can define other requirements not covered above.
+**[R104]** BPI communication protocols MUST be asynchronous.
 
-In the following, we will discuss the transaction lifecycle and its requirements as it pertains to Agreement Execution.
+**[R105]** BPI communication protocols MUST be simplex. 
 
-## 6.6 BPI Transaction Lifecycle
+For example, agent X sends a message over channel A at time T0 to agent Y. It may receive a response from Agent Y over channel B at some later time T1.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1076]: BPI User instead of agent

File: core/baseline-core-v1.0-psd01.md [hunk 11]
```diff
@@ -566,678 +637,1941 @@ Below we list and define the components of each layer. The detailed requirements
...
+  This allows unique identification of the message through its lifecycle.
 
-**[DXX]**	The Execution Framework of a VSM SHOULD have a mathematical proof of correctness and security.
+  - **[D15]**	A BPI message SHOULD contain one or more message decorators.
 
-**[RXX]**	The Execution Framework of a VSM MUST be Verifiably Secure. 
+  In general, decorators in messages at a content level allow for the support of reusable conventions that are present across multiple messages in order to handle the same functionality in a consistent manner. A relevant analogy for decorators is that they are like HTTP headers in an HTTP request. The same HTTP header is often reused as a convention across multiple requests to achieve cross-cutting functionality.
 
-**[RXX]** If a VSM can generate a valid Proof-of-Correctness for a transaction, it MUST update the state and the state history of the state object the transaction targeted based on the transaction data.
+  An initial set of useful message decorators that can be used are but are not limited to:
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1134]: `are, but`

File: core/baseline-core-v1.0-psd01.md (hunk 1 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: fixed

File: core/baseline-core-v1.0-psd01.md (hunk 2 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: updated

File: core/baseline-core-v1.0-psd01.md (hunk 4 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: fixed

File: core/baseline-core-v1.0-psd01.md (hunk 5 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: fixed

File: core/baseline-core-v1.0-psd01.md (hunk 6 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: fixed

File: core/baseline-core-v1.0-psd01.md (hunk 7 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: fixed

File: core/baseline-core-v1.0-psd01.md (hunk 8 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: done

File: core/baseline-core-v1.0-psd01.md (hunk 9 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: done

File: core/baseline-core-v1.0-psd01.md (hunk 11 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: fixed

File: core/baseline-core-v1.0-psd01.md (hunk 10 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: fixed

**Review by Consianimis — APPROVED** [review]

File: core/baseline-core-v1.0-psd01.md [hunk 12]
```diff
@@ -566,678 +652,1972 @@ Below we list and define the components of each layer. The detailed requirements
...
+  - **[R119]**	BPI Messages MUST be [JSON-LD format](#JSONLD) sympathetic.
 
-**[RXX]**	The Execution Framework of a VSM MUST ensure that state transition validation computations are either completed or abort in finite time, where what is deemed to be a suitable finite time is determined by the (commercially) allowable duration of a (commercial) transaction.
+  Note that BPI Messages may fully, and directly support JSON-LD.
 
-This requirement means that infinite computational loops cannot be allowed in a BPI. 
+## 5.5 BPI Integration
 
-**[RXX]**	The Execution Framework of a VSM MUST support widely cryptographic primitives for zero knowledge proofs natively, e.g., hashing, commitments, accumulators or zero-knowledge proof verification.
+BPI Integration or BPI Interoperability, meaning the ability of two or more BPIs to functionally interoperate with one another is both complex and crucial to generate network effects, and, thus, sustained adoption growth for the Baseline Protocol since none wants to operate in silos for long.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1175]: Review needed.

File: core/baseline-core-v1.0-psd01.md [hunk 13]
```diff
@@ -566,678 +652,1972 @@ Below we list and define the components of each layer. The detailed requirements
...
+  src="./images/10_did_auth_sign_up.png"
   >
-  <figcaption>Figure 6.2: High Level Transaction Lifecycle Flow through the BPI</figcaption>
+  <figcaption>Figure 7: Example of a Challenge-Response system based on the DIDAuth protocol using a selective disclosure response to the challenge(sdr); Source: RSKSmart at https://rsksmart.github.io/rif-identity-docs/ssi/specs/did-auth.html </figcaption>
 </figure>
 
-A prerequisite for transaction lifecycle step 4 where the BPI Processing Layer Transaction Pool pulls a transaction from BPI Middleware Layer's Messaging is that there is at least one transaction in the Messaging Capability waiting to be processed by the BPI Processing Layer.
+* Routing Layer:  A BPI is a “gated community”, hence, all BPI capabilities and BPI User services are known, or at least directly discoverable, and, therefore, directly addressable within a BPI. Therefore, there is no unknown path between sender and receiver, and, this routing is not required as it would be in a public network with an unknown number of participants of unknown identity. The scenario of a BPI User with an unknown communication endpoint will be addressed in Section 5.4, BPI Integration.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1138]: `5.4, BPI Integration` should be `5.5, BPI Integration`

File: core/baseline-core-v1.0-psd01.md [hunk 14]
```diff
@@ -566,678 +652,1972 @@ Below we list and define the components of each layer. The detailed requirements
...
+BPI Integration or BPI Interoperability, meaning the ability of two or more BPIs to functionally interoperate with one another is both complex and crucial to generate network effects, and, thus, sustained adoption growth for the Baseline Protocol since none wants to operate in silos for long. 
 
-**[DXX]**	The Execution Framework of a VSM SHOULD have a mathematical proof of correctness and security.
+This document defines BPI Interoperability as follows: BPI A and BPI B are said to be interoperable if Alice can advance the state of one or more of her state objects from BPI A by synchronizing said state object(s) with one or more of Bob's state objects on BPI B by creating a joint state between Alice and Bob without having to exit the state objects of either participant to the underlying CCSM. The exit must occur in an, ideally, censorship-resistant manner and such that neither Alice, nor Bob, nor one of their delegates can maliciously alter and finalize the state of the joint state object at any point before, during and after the state change of the joint state object on either of the participating BPIs or the (permissionless) CCSMs used by either BPIs.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1177]: Breaking to smaller sentences helps with readability.

File: core/baseline-core-v1.0-psd01.md [hunk 15]
```diff
@@ -566,678 +652,1972 @@ Below we list and define the components of each layer. The detailed requirements
...
 
-**[RXX]**	The Execution Framework of a VSM MUST be Verifiably Secure. 
+Note, that the above definition refers to East-West interoperability between BPIs. In other words the ability of Alice on BPI A to functionally interact with Bob on BPI B, such as issuing an invoice. North-South interoperability, in other words, the interoperability of components such as transaction processing or storage within a BPI is out of scope for the current version of the standard due to significant complexities.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1179]: `significant complexities.` can be deleted.

File: core/baseline-core-v1.0-psd01.md [hunk 16]
```diff
@@ -566,678 +652,1972 @@ Below we list and define the components of each layer. The detailed requirements
...
 
-**[RXX]** The Proof-of-Correctness of a state transition and associated data required for proof verification a VSM generated MUST be communicated to the CCSM Abstraction Layer for subsequent commitment to the CCSM utilized by the BPI through the Message capability in the BPI Middleware layer. 
+* Standardized set of APIs such as REST representing common asset functionalities and consisting of payloads 
+  - with defined API endpoint functionalities such as transfer, lock, unlock, exit, deliver, swap
+  - standardized API envelopes consisting of for example BPI origin and target metadata, security parameters etc. 
+  - standardized asset payloads describing the assets and their current state; a current state can consist of for example (zero-knowledge) proof set (state-object-history, state-object-locks, state-object-state) and state object description can consist of for example state-object type, state-object ID, state-object owner, anchor contract(s) for proof verification(s) -- See also related work here 1 -- are critical to have the same “words and grammar” to be able to talk to one another.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1194]: missing link `-- See also related work here 1 --`

File: core/baseline-core-v1.0-psd01.md [hunk 17]
```diff
@@ -566,678 +652,1972 @@ Below we list and define the components of each layer. The detailed requirements
...
 
-Note, see the BPI transaction lifecycle management flow in Fig 6.2, in particular step (8).
+* Discoverable Standard Transport security such as JOSE with JWS/JWE or DIDComm is critical to ensure security and privacy at all times and beyond HTTPS. See also related work here.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1196]: missing link `See also related work here.`

File: core/baseline-core-v1.0-psd01.md [hunk 18]
```diff
@@ -566,678 +652,1972 @@ Below we list and define the components of each layer. The detailed requirements
...
+Before outlining the requirements for BPI Interoperability this document wants to provide a concrete, detailed, and real world non-normative example of BPI Interoperability because the technical requirements to implement such interoperability are quite complex.
 
-**[OXX7]** A BPI MAY consist of more than one processing node.
+One of the most common processes occurring every day across the world is the importing of goods into a country and to be delivered to a customer in that country. This involves a shipper of the goods, Alice, an importer of the goods, Bob, and a country customs organization, Claire.
 
-We will call this a BPI network.
+This scenario has, therefore, 3 workflows intersecting which are independent yet related to one another: Alice has her shipping workflow of containers coming into a harbor with goods that Bob with his importing workflow wants to pass through Claire's custom workflow to be able to deliver it to his customer. Note, that because of reg
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1202]: `Note, that because of reg` can be removed?

File: core/baseline-core-v1.0-psd01.md [hunk 19]
```diff
@@ -566,678 +652,1972 @@ Below we list and define the components of each layer. The detailed requirements
...
+**Example BPI Interoperability Workflow:**
 
-**[DXX]** All other data SHOULD be stored as fully persistent data in the Storage capability.
+1. Alice locks the state of her current workflow workstep on her BPI
 
-This is a consequence of step (7) in Fig. 6.2 above.
+2. Alice opens a state channel committing an initial state consisting of Set A
 
-The Storage capability must also meet all requirements defined in [Section 7](#7-General-BPI-Storage-Capabilities)
+    Note, besides a state channel any singleton state machine such as a sidechain or other Layer 2 solotion may be employed.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1232]: solution

File: core/baseline-core-v1.0-psd01.md [hunk 20]
```diff
@@ -566,678 +652,1972 @@ Below we list and define the components of each layer. The detailed requirements
...
 
--------
+3. Alice sends state channel invites to Bob and Claire based on their communication endpoints in their DID doc creating the equivalent of a temporary workgroup. The invite must include the state information both Bob and Clair need to submit to the state channel, Set B and et C, and the required agreement criteria e.g. 2 of 3 digital signatures over the joined state of the ZK proofs in the form of an agreement predicate. The predicate would be a zk prover circuit with specified private and public inputs that can be run by Bob and Claire to generate zk proofs proving correct state updates. In this case, simple signatures over the zk-proofs are chosen.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1234]: `et C` to `set C`

**Review by humbitious — APPROVED** [review]

**Pull request merged**